clerk.go

```diff
@@ -388,12 +389,12 @@ func (params ListParams) ToQuery() url.Values {
 	return q
 }
 
-// Regular expression that matches multiple backslashes in a row.
-var extraBackslashesRE = regexp.MustCompile("([^:])//+")
-
 // JoinPath returns a URL string with the provided path elements joined
 // with the base path.
 func JoinPath(base string, elem ...string) (string, error) {
+	// Regular expression that matches multiple backslashes in a row.
+	var extraBackslashesRE = regexp.MustCompile("([^:])//+")
```

Comment:
❓ This change will cause the regular expression to recompile, with a slight overhead. I think the previous version was fine, did you find a problem with it?

Reply:
No, I was experimenting a bit in a WIP branch and messed it up. Thanks for noticing.